core-httpclient-impl/src/main/java/com/optimizely/ab/OptimizelyFactory.java

```diff
@@ -319,10 +322,16 @@ public static Optimizely newDefaultInstance(ProjectConfigManager configManager,
             .withNotificationCenter(notificationCenter)
             .build();
 
+        ODPApiManager defaultODPApiManager = new DefaultODPApiManager();
+        ODPManager odpManager = ODPManager.builder()
+            .withApiManager(defaultODPApiManager)
```

Comment:
We cannot do that because DefaultODPApiManager implements http calls and is only available in the `http-client` project. The Core SDK does not know about its implementation. This is exactly how we inject polling datafile manager and batch event processor.